src/esn.calendar.libs/app/services/cached-event-source.js

```diff
@@ -79,6 +79,12 @@ require('./cal-ui-authorization-service.js');
             change.instances = [];
             change.event.expand(start.clone().subtract(1, 'day'), end.clone().add(1, 'day')).forEach(function(subEvent) {
               if (subEvent.status !== 'CANCELLED') {
+
```

Comment:
```suggestion
```

Reply:
this is not resolved, the blank line is unnecessary.

check: https://arcticicestudio.github.io/styleguide-javascript/rules/whitespace.html#padded-blocks

Reply:
it is resolved in my second commit